tutorial/source/custom_objectives.ipynb

```diff
@@ -214,32 +214,25 @@
     "svi.step(other_args, annealing_factor=0.2, latents_to_anneal=[\"my_latent\"])\n",
     "```"
    ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": []
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 2",
    "language": "python",
-   "name": "python3"
+   "name": "python2"
   },
   "language_info": {
    "codemirror_mode": {
     "name": "ipython",
-    "version": 3
+    "version": 2
    },
    "file_extension": ".py",
    "mimetype": "text/x-python",
    "name": "python",
    "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.6.2"
+   "pygments_lexer": "ipython2",
+   "version": "2.7.13"
```

Comment:
can you revert this by running `make scrub` or changing your ipython kernel to 3.6?

Reply:
i scrubbed should be ok to merge